pkgs/top-level/aliases.nix

```diff
@@ -303,8 +303,8 @@ mapAliases ({
   rssglx = rss-glx; #added 2015-03-25
   ruby_2_0_0 = throw "deprecated 2018-0213: use a newer version of ruby";
   ruby_2_1_0 = throw "deprecated 2018-0213: use a newer version of ruby";
-  ruby_2_2_9 = throw "deprecated 2018-0213: use ruby_2_2 instead";
-  ruby_2_3_6 = throw "deprecated 2018-0213: use ruby_2_3 instead";
```

Comment:
can you also add ruby_2_3 in there?

Reply:
I didn’t because we don’t generally do it when removing packages IIRC, and I don’t think aliases.nix should have to contain a list of every attribute to have ever existed.

But it does look like we’ve started here anyway, so I can do, yeah.

Reply:
Yeah any removal should be added to this list. And at some point the list will be pruned. This is to allow users that upgrade to get a meaningful error message.